src/spec-lang/expr_grammar.pegjs

```diff
@@ -11,7 +11,8 @@ Annotation =
         / UserFunctionDefinition
```

Comment:
In the Annotation rule, shouldn't the `prefix` now be reqiured? I.e. it should be:

```
          prefix: '#'
```

Reply:
`#` is no longer required due to https://github.com/ConsenSys/scribble/pull/96/commits/e57764f3acdbedbb98e4e3d041446f2210f42531 introduces `macro` keyword.